networks/p2p/peer_test.go

```diff
@@ -154,7 +154,7 @@ func TestPeerDisconnect(t *testing.T) {
 // This test is supposed to verify that Peer can reliably handle
 // multiple causes of disconnection occurring at the same time.
 func TestPeerDisconnectRace(t *testing.T) {
-	maybe := func() bool { return rand.Intn(1) == 1 }
+	maybe := func() bool { return rand.Intn(2) == 1 }
```

Comment:
rand.intn(1) returns 0 or 1.
Please refer to https://golang.org/pkg/math/rand/#Intn

Reply:
Sorry. It returns 0 always becaus it returns [0,1).
Please ignore this comment. :)

Reply:
`rand.intn(1)` always returns 0. It open about `n`. 

`Mathematical interval notation such as [0, n) is used throughout the documentation for this package.` by https://golang.org/pkg/math/rand/#Intn